README.md

```diff
@@ -296,14 +298,25 @@ Or the following `*.jsonschema.json` files, when the `strict` option is set to `
 
 The JSON Schema plugin supports the following options:
 
+- `bundle` - One of `true`, `false`, or `all`. Defaults to `all`. If `true`, only the bundle
```

Comment:
Having true/false as well as all is confusing. How about a separate option `bundle_only=true|false` (default: false).

Reply:
I want the default to generate all the options, as this is what the BSR hosts by default

Reply:
That wouldn't change the defaults though, right? You'd have `bundle=true|false` (default: true) and `bundle_only=true|false` (default: false). That would match the all option semantics.

Reply:
the default is all

Reply:
Oh, you want the literal "true|false" instead of "all" (the order doesn't matter)

Reply:
Switched to just a list of targets